projects/aws/lib/step-function.ts

```diff
@@ -0,0 +1,113 @@
+import cdk = require('@aws-cdk/core')
+import lambda = require('@aws-cdk/aws-lambda')
+import { Code, FunctionProps } from '@aws-cdk/aws-lambda'
+import s3 = require('@aws-cdk/aws-s3')
+import iam = require('@aws-cdk/aws-iam')
+import sfn = require('@aws-cdk/aws-stepfunctions')
+import tasks = require('@aws-cdk/aws-stepfunctions-tasks')
+import { toTitleCase } from './tools'
+import { Duration } from '@aws-cdk/core'
+interface StepFunctionProps {
+    stack: string
+    stage: string
+    deployBucket: s3.IBucket
+    publishedBucket: s3.IBucket
+    backendURL: string
+}
+
+const taskλ = (
+    name: string,
+    {
+        scope,
+        stack,
+        stage,
+        deployBucket,
+        publishedBucket,
+    }: {
+        scope: cdk.Construct
+        stack: string
+        stage: string
+        deployBucket: s3.IBucket
+        publishedBucket: s3.IBucket
+    },
+    environment?: { [key: string]: string },
+    overrides?: Partial<FunctionProps>,
+) => {
+    return new lambda.Function(scope, `EditionsArchiver${toTitleCase(name)}`, {
+        ...{
```

Comment:
Any reason this isn't part of the root object, rather than spreading it in? Confused me for a second.

Reply:
Nope, good call.